app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RatingsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+  end
+
+  def new
+    @rating = Rating.new
+  end
+
+  def create
+    @rating = @rateable.ratings.new(rating_params)
+    @rating.user = current_user
+    reviewer = Review.find_by(id: @rating.rateable_id).user
```

Comment:
If you know that rateable is definitely a Review here (do you?) then you can just call rating.rateable.user - you don't need this query